charts/ratify/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: ratify
 description: A Helm chart for Ratify
-version: 1.14.0
+version: 1.14.1
```

Comment:
Hi @mannbiher , we don't currently have a streamline process to release chart without binary release, please keep the version at 1.14.0. Please use your local chart for now, thanks for understanding.

Reply:
Done.